Xamarin.Forms.Platform.UAP/LabelRenderer.cs

```diff
@@ -128,6 +131,7 @@ protected override void OnElementChanged(ElementChangedEventArgs<Label> e)
 				_isInitiallyDefault = Element.IsDefault();
 
 				UpdateText(Control);
+				UpdateTextDecorations();
```

Comment:
I know this isn't consistent between platforms :-/ but can you change this so Control is passed in here? Since that's at least how all the other updates are here on UWP